src/test/netbase_tests.cpp

```diff
@@ -31,9 +31,7 @@ static CNetAddr ResolveIP(const std::string& ip)
 
 static CSubNet ResolveSubNet(const std::string& subnet)
 {
-    CSubNet ret;
-    LookupSubNet(subnet, ret);
-    return ret;
+    return LookupSubNet(subnet);
 }
```

Comment:
Now `ResolveSubNet()` is identical to `LookupSubNet()`. Maybe ditch the former and change the callers to use directly `LookupSubNet()`?

Reply:
Yes, better to call `LookupSubNet()`.